web/src/beta/features/Visualizer/shared/hooks/useTimelineBlock.ts

```diff
@@ -43,14 +44,19 @@ export default (timelineValues?: TimelineValues) => {
 
   const playSpeedOptions = useMemo(() => {
     const speedOpt = [
-      "1sec/sec",
-      "0.5min/sec",
-      "1min/sec",
-      "0.1hr/sec",
-      "0.5hr/sec",
-      "1hr/sec"
+      {
+        timeString: "Not set",
+        seconds: 0
+      },
+      { timeString: "1sec/sec", seconds: 1 },
+      { timeString: "0.5min/sec", seconds: 30 },
+      { timeString: "1min/sec", seconds: 60 },
+      { timeString: "0.1hr/sec", seconds: 360 },
+      { timeString: "0.5hr/sec", seconds: 1800 },
+      { timeString: "1hr/sec", seconds: 3600 }
     ];
-    return convertOptionToSeconds(speedOpt);
+
+    return speedOpt;
```

Comment:
We can define this `playSpeedOptions` out of the hooks as const directly, no need to use useMemo